frontend/src/scenes/dashboard/newDashboardLogic.ts

```diff
@@ -169,5 +175,28 @@ export const newDashboardLogic = kea<newDashboardLogicType>([
             }
             actions.setIsLoading(false)
         },
+        [dashboardTemplatesLogic.actionTypes.getAllTemplatesSuccess]: ({ allTemplates }) => {
```

Comment:
I might be being picky but this flow feels too implicit

If I'm understanding

* We're mounting the `dashboardTemplateLogic` which will cause `getAllTemplates` to fire
* add listening to `getAllTemplatesSuccess` and looking up the template when it fires.

I'd have expected `dashboardTemplateLogic` to let us look up a template by slug. So `newDashboardLogic` would have a `urlToAction` that triggered a local action that used that connected `dashboardTemplateLogic` action (if that confusing sentence makes sense 🤣 )